pkg/services/user/identity.go

```diff
@@ -27,7 +27,9 @@ type SignedInUser struct {
 	// AuthID will be set if user signed in using external method
 	AuthID string
 	// AuthenticatedBy be set if user signed in using external method
-	AuthenticatedBy  string
+	AuthenticatedBy            string
+	AllowedKubernetesNamespace string
```

Comment:
Nit: Should we just use namespace here like he do in our tokens? If so we should probably document that this is mapped to k8s namespace

Reply:
This was named after talking with Ryan on my team. There is already named properties with the name namespace in them, that refer to the auth/identity namespace here, so that trying to demarcate precisely.